test/runnable/test18902.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$DMD -m${MODEL} -lib 18902.a || test $? = 1
```

Comment:
This doesn't check for the exit code.
If DMD exists with 0, it still will pass.
I recommend the following:
- use grep to check for the error message (check the other bash tests for examples)
- use `! $DMD` to ensure that only a failing DMD exit code will pass the test

Reply:
I don't see a need to check the error message since the bug can be identified with the return code.
What about:
```d
if $DMD -m${MODEL} -lib 18902.a; then
	exit 1
else
	test $? = 1
fi
```
?
But if `grep` is a preferd way for this kind of tests, I can rewrite it with grep, it doesn't matter that much.

Reply:
I agree with Sebastian, you want to check for an error message and exit code 1, so that both cannot regress.

Reply:
But the modified code does `exit 1` and exits with a non-zero exit code if the result becomes zero.